lib/Interpreter/CIFactory.cpp

```diff
@@ -928,29 +961,132 @@ namespace {
   }
 
   static void AddRuntimeIncludePaths(llvm::StringRef ClingBin,
-                                     clang::HeaderSearchOptions& HOpts) {
-    if (HOpts.Verbose)
-      llvm::errs() << "Adding runtime include paths:\n";
-    // Add configuration paths to interpreter's include files.
+                                    std::vector<const char*>& args,
+                                    bool Verbose) {
+    static std::vector<std::string> sArguments;
```

Comment:
Could you implement that without statics? Statics are troublesome for multi interpreter setups.